retrofit/src/main/java/retrofit/Types.java

```diff
@@ -32,8 +29,6 @@
  * @author Jesse Wilson
```

Comment:
Delete this while you're at it? Deleting `@author` lines in general is bad (cause you're denying somebody credit for their work) but personally I'd prefer to not be named.